highfidelity/hifi-content
# updated to create regular churros and crunch.js sound location

## Test Plan
1. Go to hifi://mexico/506.479,498.46,496.46/0,-0.743432,0,0.668812 (in front of churro stand)
2. Click on churro stand
3. A churro should appear
4. Drag the churro to your mouth
5. You should hear a crunch and the churro will disappear

Hopefully fixes https://highfidelity.fogbugz.com/f/cases/13902/Mystery-Eating-crunching-Churro-sounds-in-Mexico-near-fountain (won't know until more bug reports come up)
